mypy/lex.py

```diff
@@ -740,15 +747,18 @@ def lex_break(self) -> None:
             last_tok.string += self.pre_whitespace + s
             self.i += len(s)
             self.line += 1
+            self.column = 0
```

Comment:
so we're incrementing the line here, so have to reset column back to 0.
in this class, `self.column` is keeping track of the column we're at (i.e. within the line) whereas `self.i` keeps track of the position within the overall string.